test_image.sh

```diff
@@ -223,6 +225,39 @@ function test_docker_image_scripts_2 {
 	test_page "http://${CONTAINER_HOSTNAME}:${CONTAINER_HTTP_PORT}/test_docker_image_scripts_2.jsp" "TEST2"
 }
 
+function test_docker_image_with_playwright {
+	echo ""
+	echo "Waiting for application to be fully ready..."
+
+	while true
+	do
+		if docker logs "${CONTAINER_ID}" 2>&1 | grep -q "License registered for DXP Development" #name is not right at all
+		then
+			echo "Application is fully ready!"
+
+			break
+		fi
+
+		echo -n "."
+		sleep 3
+	done
+
+	export CONTAINER_PORT_EXPORTED=$(docker port ${CONTAINER_ID} 8080 | head -n 1 | awk -F: '{print $2}')
+
+	cd playwright-dependencies
```

Comment:
@lucasmiranda0, please check if the `lc_cd` is being used in this file, if so please use this instead. Otherwise, make sure this folder exists before try to enter on it.